go/vt/sqlparser/ast.go

```diff
@@ -5393,14 +5393,19 @@ type GroupConcatExpr struct {
 	Distinct  string
 	Exprs     SelectExprs
 	OrderBy   OrderBy
-	Separator string
+	Separator Separator
+}
+
+type Separator struct {
```

Comment:
Would be nice to give this struct some godocs and describe its intent. We don't always do a great job with docs in this package, but no reason not to start with this one and make it really easy for future maintainers to understand. 😄